client/src/components/modals/payment-request-modal/payment-request-modal.component.tsx

```diff
@@ -11,32 +11,39 @@ import { AddressId } from '../../../models/generated/addressId';
 
 import { CrossIcon } from '../../common/icons/common.icons';
 
+import './payment-request-modal.scss';
+
 type IPaymentRequestModalProps = RouteComponentProps<any> & {
   isOpen: boolean;
   onClose: () => void;
   address: AddressId;
 };
 
-import './payment-request-modal.scss';
+interface IPaymentRequestState {
+  amount: number;
+  copied: boolean;
+  description: string;
+}
+
+class PaymentRequestModal extends React.PureComponent<IPaymentRequestModalProps, IPaymentRequestState> {
 
-class PaymentRequestModal extends React.PureComponent<IPaymentRequestModalProps> {
   link: HTMLDivElement;
   
-  state: any = {
+  state: IPaymentRequestState = {
     amount: 0,
     copied: false,
     description: ''
   };
   
-  constructor (props: any) {
-    super(props);
-    
-    this.setAmount           = this.setAmount.bind(this);
-    this.setDescription      = this.setDescription.bind(this);
-    this.copyLinkToClipboard = this.copyLinkToClipboard.bind(this);
-    this.selectLink          = this.selectLink.bind(this);
+  constructor (props: IPaymentRequestModalProps) {	
+    super(props);	
```

Comment:
Remove whitespaces please, I use WebStorm ( IDEA ) for formatting my code, I will add codestyle XML for webstorm later to the project